gnd/src/main/java/com/google/android/gnd/ui/field/FieldModel.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.ui.field;
+
+import android.content.res.Resources;
+import androidx.databinding.BaseObservable;
+import androidx.databinding.Bindable;
+import com.google.android.gnd.BR;
+import com.google.android.gnd.R;
+import com.google.android.gnd.model.form.Field;
+import com.google.android.gnd.ui.common.Validator;
+
+/**
+ * {@link FieldModel} is not shared among {@link FieldView}. It stores the current state of the View
+ * it's attached to and updates itself using two-way data binding.
+ */
+public class FieldModel extends BaseObservable {
+
+  private final Resources resources;
+  private final Field field;
+
+  private String response;
+  private String error;
+
+  FieldModel(Field field, Resources resources) {
+    this.field = field;
+    this.resources = resources;
+  }
+
+  @Bindable
+  public String getResponse() {
+    return response == null ? "" : response;
+  }
+
+  public void setResponse(String response) {
```

Comment:
Since this does more than sets the response I'd suggest using another name (updateResponse?)